tests/ruff.toml

```diff
@@ -10,6 +10,8 @@ extend-ignore = [
     "TCH001",
     # https://beta.ruff.rs/docs/rules/#flake8-tidy-imports-tid
     "TID251",  # banned-api - is used to prevent opaque attribute errors caused by missing optional dependencies
+    # https://beta.ruff.rs/docs/rules/#flake8-bugbear-b
+    "B011", # assert-false - common pattern in pytest
```

Comment:
Disabling this rule for tests because `assert False` is a common thing to do in tests to force a failure.